staticd/static_vty.c

```diff
@@ -815,23 +531,22 @@ DEFPY(ip_route_blackhole_vrf,
       "Table to configure\n"
       "The table number to configure\n")
 {
-	VTY_DECLVAR_CONTEXT(vrf, vrf);
-	struct static_vrf *svrf = vrf->info;
-
-	if (table_str && !vrf_is_backend_netns()) {
-		vty_out(vty,
-			"%% table param only available when running on netns-based vrfs\n");
-		return CMD_WARNING_CONFIG_FAILED;
-	}
+	const struct lyd_node *vrf_dnode;
+	const char *vrfname;
 
+	vrf_dnode =
+		yang_dnode_get(vty->candidate_config->dnode, VTY_CURR_XPATH);
+	if (!vrf_dnode)
+		return CMD_SUCCESS;
```

Comment:
Please return `CMD_WARNING_CONFIG_FAILED` instead (also applies to the other commands where this can happen).

Reply:
done